### ISSUE TITLE
Nothing is returned for `outputs.url` in v1

### ISSUE THREAD

**Issue by nwaughachukwuma** [google-github-actions/deploy-appengine#288]
### TL;DR

We are currently upgrading all our actions from `google-github-actions` to v1, after the release. Everything works well so far but `deploy-appengine` fails with an error that says `the url is malformed` when `outputs.url` is accessed from subsequent steps.

Attempting to print the url to stdout gave a blank result which shouldn't be since the [readme](https://github.com/google-github-actions/deploy-appengine#outputs) specifies that the url remains an output.

### Expected behavior

`outputs.url` should be well formed, i.e., it should be the version url whether `--promote` or `--no-promote`

### Observed behavior

`outputs.url` is malformed, and an attempt to echo to stdout gave blank output

### Action YAML

```yaml
deploy:
    timeout-minutes: 10
    runs-on: self-hosted
    steps:
      - uses: actions/setup-node@v3
        with:
          node-version: 16
      - uses: actions/download-artifact@v3
        with:
          name: ${{ inputs.PROJECT }}
          path: ${{ inputs.WORKING_DIRECTORY }}
      - uses: google-github-actions/auth@v1
        with:
          credentials_json: ${{ secrets.SERVICE_ACCOUNT_KEY }}
      - uses: google-github-actions/setup-gcloud@v1
      - id: deploy
        uses: google-github-actions/deploy-appengine@v1
        with:
          project_id: ${{ inputs.PROJECT_ID }}
          deliverables: ${{ inputs.CONFIG_FILE }}
          promote: ${{ inputs.PROMOTE }}
          version: ${{ inputs.VERSION }}
          working_directory: ${{ inputs.WORKING_DIRECTORY }}
      - name: health_check
        run: curl -f "${{ steps.deploy.outputs.url }}{{ inputs.HEALTH_CHECK_PATH }}"
```

### Log output

```text
Run curl -f "/admin"
[14](https://github.com/.../actions/runs/3702097274/jobs/6272090675#step:10:15)
curl: (3) <url> malformed
[15](https://github.com/.../actions/runs/3702097274/jobs/6272090675#step:10:16)
Error: Process completed with exit code 3.
```

**Comment by sethvargo** [comment]
Are you saying this _was_ working in v0?

**Comment by nwaughachukwuma** [comment]
> but are you using --no-promote

Yes for versions deployed from PRs.

> Are you saying this was working in v0?

Yes it worked in `v0`

I see you checked-in a fix in #290. Waiting for the release.

**Comment by sethvargo** [comment]
Done

**Comment by nwaughachukwuma** [comment]
Awesome, thanks!
